xds/src/test/java/io/grpc/xds/XdsClientWrapperForServerSdsTest.java

```diff
@@ -163,7 +164,8 @@ static void generateListenerUpdateToWatcher(
   @Before
   public void setUp() throws IOException {
     MockitoAnnotations.initMocks(this);
-    xdsClientWrapperForServerSds = new XdsClientWrapperForServerSds(PORT, xdsClient, null);
+    xdsClientWrapperForServerSds = new XdsClientWrapperForServerSds(PORT);
+    xdsClientWrapperForServerSds.start(xdsClient);
```

Comment:
can you cleanup in @After.

Reply:
`start(xdsClient)` just sets up a watch but in this case `xdsClient' is supposed to be a mock so there is nothing to clean up here. Agree?

Reply:
i think `timeServiceResource` still need to be cleaned.

Reply:
`timeServiceResource` is not touched in the tests which call only `start(xdsClient)` . Mainline code calls `start()` which uses `timeServiceResource`. Pls verify

Reply:
okay it is not used. but i don't understand why you don't want to add cleanup? what's the downside? i can't think any downsides other than few positives.

Reply:
There is no downside. What is there to call in cleanup? Just call `xdsClientWrapperForServerSds.shutdown()` knowing that it will have no effect? I am asking because having superfluous code can also look bad

Reply:
maybe it is just for me, but calling start and not calling shutdown look bad. it almost feel like you don't call close for closeable because you maybe know the internal at that time.

Reply:
I don't mind adding the shutdown calls and I will add them. But I can see someone commenting that this is superfluous code. And I remember seeing those comments before.

Reply:
if your api design is calling shutdown only when it must be, i am fine with what you are saying.

Reply:
added